### ISSUE TITLE
[balls] A simple int arg for health and attack bonus in /count and /admin count

### ISSUE THREAD

**Issue by ghost** [laggron42/BallsDex-DiscordBot#461]
**Describe the feature you'd like**
You would be able to type a number you want for the attack bonus or health bonus, and it would count the balls you have with that bonus only.

**Additional context**
It would need checks to prevent numbers more than the settings.max_attack and settings.max_health.

**Comment by imtherealF1** [comment]
Thinking about it again, I don't think the arguments will be necessary for /admin count. I already made the code for /balls count ready. and I want to hear opinions on whether I should also make it for /admin count or not.

**Comment by flaree** [comment]
Personally i'm not a fan, I just think its more clutter for the arguments

**Comment by imtherealF1** [comment]
I think that /count should have some sorting options, and it only has 3 which isn't cluttered at all, and with the shinies being moved to special soon it will become 2, so I think that it wouldn't be a big issue.

**Comment by laggron42** [comment]
I agree with flare where we should avoid too much cluttering, but at the same time advanced filtering is useful. I've been considering a pseudo language for advanced filtering like this, may want to wait for this
